### PR TITLE
Running Alerts colorized depending on to their severity

### PR DESCRIPTION
**Pull request by SourceDoctor**
similar to Alert Widget, Colors should symbolize their severity for active Alerts

image

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by kkrumm1** [comment]
Looks great!

File: includes/html/print-alert-rules.php [hunk 1]
```diff
@@ -200,9 +201,27 @@
             $status_msg = "All devices matching " . $rule['name'] . "  are OK";
         }
         if ((int) $sub['state'] === 1 || (int) $sub['state'] === 2) {
-            $ico   = 'exclamation';
-            $col   = 'danger';
-            $extra = 'danger';
+            switch ($severity) {
```
Comment (Jellyfrog) [inline includes/html/print-alert-rules.php]: This should probably be a generic function, so we use the same classes everywhere

Comment (SourceDoctor) [reply includes/html/print-alert-rules.php]: done :-)

**Comment by Jellyfrog** [comment]
There are a couple of places that could be combined probably;
https://github.com/librenms/librenms/blob/6dfa5ed75a9ec0f8dd8cdd0fe32aaf1f6501ea0a/includes/html/table/alerts.inc.php#L135-L148

**Comment by SourceDoctor** [comment]
@Jellyfrog 
maybe centralize them in an extra PR?

**Comment by SourceDoctor** [comment]
@Jellyfrog 
done :)

**Comment by laf** [comment]
This is just the alert rules page though isn't it. Imho I don't think we need to colour each row based on the severity the rule is set at.

**Comment by SourceDoctor** [comment]
@laf 
This Alert Rule Page is also used in edit Device context. Filtered down to Alert Rules matching to the Device you are in.
It's for nice and fast seeing which Rule is actually alarming and which severity it has.
In Alert Widget Severity also has specific colors, so why in Alert Rules only read (which means critical)?

**Comment by SourceDoctor** [comment]
@Jellyfrog 
something else, or ready for merge?

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-61-release-changelog-february-2020/11145/1

**Pull request merged**